poker-hand-evaluator/src/evaluator/Player.java

```diff
@@ -4,44 +4,49 @@
 import java.util.List;
 
 public class Player {
-    private String name;
-    private List<Card> playCardList;
+    private final String name;
+    private final List<Card> playerCardList;
```

Comment:
seems odd to me, since a player has a different list of cards in each round - am I going to create a new player in each round or am I going to empty the list of cards each time? Both are doable - but both solutions are code smells, because the structure is odd

Reply:
I just follow the flow of the game: shuffle, deal, evaluate, sweep...
How will I declare a winner at the end of the round if I don't link the card lists to the players?
How to do it easier?